<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_FIle"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuE">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuW">
    <property name="title">
     <string>Window(&amp;W)</string>
    </property>
    <addaction name="actionClose_O"/>
    <addaction name="actionClose_All_A"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrev"/>
   </widget>
   <addaction name="menuF"/>
   <addaction name="menuE"/>
   <addaction name="menuW"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>New File(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_FIle">
   <property name="text">
    <string>Open FIle(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As(&amp;A)...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close(X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionClose_O">
   <property name="text">
    <string>Close(&amp;O)</string>
   </property>
  </action>
  <action name="actionClose_All_A">
   <property name="text">
    <string>Close All(&amp;A)</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev(V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
